setup.py

```diff
@@ -10,18 +10,26 @@
 import subprocess
 import sys
 
-import subprocess
-import os
-
 from setuptools import Command
 from skbuild import setup
 
+"""
+A generic wrapper to access nmodl binaries from a python installation
+Please create a softlink with the binary name to be called.
+"""
+import stat
+from pkg_resources import working_set
+from pywheel.shim.find_libpython import find_libpython
+
 
 # Main source of the version. Dont rename, used by Cmake
```

Comment:
Just to be sure: from the subject topic, is this related change?

Reply:
this is due to "blacking" setup.py. I just reordered the code around